galaxy_ng/app/access_control/mixins.py

```diff
@@ -18,6 +18,8 @@ class GroupModelPermissionsMixin:
 
     @property
     def groups(self):
+        from pulpcore.plugin.util import get_groups_with_perms_attached_roles
```

Comment:
Do all of these deferred imports need to be here with the changes to pulp container?

Reply:
Probably.